search/search-dev-tools/src/style/prism.scss

```diff
@@ -14,7 +14,7 @@
 		white-space: pre;
 		word-spacing: normal;
 		word-break: normal;
-		font-size: 0.9em;
+		font-size: 12px;
```

Comment:
Maybe we should try to find the proper `em` value? `em`s scale better if the user wants to zoom in or zoom out